tests/utils/LinearAlg2DTest.cpp

```diff
@@ -136,57 +133,63 @@ TEST_P(GetAngleTest, GetAngle)
     const double angle = angle_degrees * M_PI / 180.0;
 
     const double supposed_angle = LinearAlg2D::getAngleLeft(a, b, c);
-    ASSERT_LE(std::fabs(angle - supposed_angle), maximum_error) << "Corner in " << a << " - " << b << " - " << c << " was computed to have an angle of " << supposed_angle << " instead of " << angle << ".";
+    ASSERT_LE(std::fabs(angle - supposed_angle), maximum_error)
+      << "Corner in " << a << " - " << b << " - " << c << " was computed to have an angle of " << supposed_angle << " instead of " << angle
+      << ".";
```

Comment:
The period on a separate line?

Reply:
once the indent is 4 space there will probably be more on the second line